File: Java/Recursion-Programs/46-largest-element-in-array.java
class Main {
    static int Largest(int[] arr, int larg, int index){
        if (index == arr.length)  {
            return larg;
        }
        if (arr[index] > larg){
            larg = arr[index];
        }
        return Largest(arr, larg, index+1);
    }

    public static void main(String[] args) {
        int[] arr = {10, 25, 5, 75, 15};

        System.out.println("Largest value is: " + Largest(arr, arr[0], 0));
    }
}

File: Python/3-Recursion-Programs/46-largest-element-in-array.py
arr = [10,20,5,75,15]

def Largest(arr, large, index):
    if index == len(arr):
        return large
    
    if(arr[index] > large):
        large= arr[index]

    return Largest(arr, large, index+1)

print("Largest value is:", Largest(arr, arr[0], 0))